wallet/src/actors/app/state.rs

```diff
@@ -56,19 +56,28 @@ impl State {
         }
     }
 
+    /// Get all wallets for a session
+    pub fn wallets(
```

Comment:
```suggestion
    pub fn session_wallets(
```

Maybe something more descriptive?